courses/serializers.py

```diff
@@ -201,3 +202,38 @@ class Meta:
             'requirements',
             'sections',
         ]
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = [
+            'username',
+            'first_name',
+            'last_name',
+        ]
+
+
+class UserDataSerializer(serializers.ModelSerializer):
+    user = UserSerializer(read_only=True)
+
+    class Meta:
+        model = UserData
+        fields = [
+            'email',
+            'phone',
+            'user'
+        ]
+
+
+class StatusUpdateSerializer(serializers.ModelSerializer):
+    section = serializers.ReadOnlyField(source='section__full_code')
+
+    class Meta:
+        model = StatusUpdate
+        fields = ['section', 'old_status', 'new_status', 'created_at', 'alert_sent', 'request_body']
```

Comment:
`request_body` doesn't neeed to be included in the serializer. it was mostly for debugging purposes when we were building the webhook